src/plugins/vis_types/timeseries/server/lib/vis_data/response_processors/series/math.test.js

```diff
@@ -253,7 +253,7 @@ describe('math(resp, panel, series)', () => {
       )(await mathAgg(resp, panel, series)((results) => results))([]);
     } catch (e) {
       expect(e.message).toEqual(
-        'Failed to parse expression. Expected "*", "+", "-", "/", end of input, or whitespace but "(" found.'
+        'Failed to parse expression. Expected "*", "+", "-", "/", "<", "=", ">", end of input, or whitespace but "(" found.'
```

Comment:
nit - should be `==` now

Reply:
The message is correct. The grammar picks the first possible valid char at the time.
If the expression becomes `divide = params.a, params.b` it will error with `Expected "=" but " " found` as it's the only valid route.